website/source/docs/builders/vmware-iso.html.md

```diff
@@ -28,6 +28,25 @@ scratch, booting it, installing an OS, provisioning software within the OS, then
 shutting it down. The result of the VMware builder is a directory containing all
 the files necessary to run the virtual machine.
 
+### VMware Player on Linux
+
+There are (as of 2016-Apr-18) two versions of VMware Player currently available for download:
+
+* [7.0](https://my.vmware.com/en/web/vmware/free#desktop_end_user_computing/vmware_player/7_0)
+* [12.0](https://my.vmware.com/en/web/vmware/free#desktop_end_user_computing/vmware_workstation_player/12_0)
```

Comment:
The problem is that the different versions give you different capabilities.

For example, later versions of vSphere will happily virtualise a VM using VMware player 7 but not so much player 12.

I'll try and re-work things to default to the latest (12) version even though it is more complicated to get going. The most common use case is a local laptop rather than a VM running within VM trying to run VMware player.